README.md

```diff
@@ -24,21 +24,27 @@ require "crystalla"
 # Matrixes are created from column arrays
 m = Crystalla::Matrix.columns [[1.0, 2.0], [3.0, 4.0]]
 
-p "This is how M looks now: "
-m.print
+puts "This is how M looks now: "
```

Comment:
It's not a preference, they do different things. `puts` invokes `to_s`, `p` invokes `inspect`. In the case of a String, `inspect` shows quotes and escapes like "\n" and "\t". So doing `p "Hello"` will print `"Hello"` instead of `Hello`, which is probably not what you want the user to see.